packages/core/src/cli/commands.ts

```diff
@@ -88,21 +91,13 @@ export function runCli(): void {
     .action(async (options: InspectOptions) => {
       try {
         // TODO: inspect should output Rslib's config
-        const rslibConfig = await loadConfig({
-          path: options.config,
-          envMode: options.envMode,
-        });
-        const environments = await composeRsbuildEnvironments(rslibConfig);
-        const rsbuildInstance = await createRsbuild({
-          rsbuildConfig: {
-            environments: pruneEnvironments(environments, options.lib),
-          },
-        });
-        await rsbuildInstance.inspectConfig({
+        const { root, rslibConfig } = await getRslibConfig(options);
```

Comment:
Environment variables may not be passed in during runtime since user may use js api to start the child process without inherit some env. 

The reason for passing root is to read tsconfig and package.json. I think we can extract the relevant logic to the outside when we doing configuration verification and no configuration file mode in the future.

Reply:
> And also we will design our javascript API, so the root can be put into a context to pass around.

Ok, good, thanks for explaining. Do you already have a ticket for this?

Reply:
We are discussing that we should follow Rsbuild to resolve package.json and tsconfig.json, I think the package.json should be in folder as same as webpack context. https://rsbuild.dev/config/root#root